addons/website_links/static/src/xml/recent_link.xml

```diff
@@ -2,39 +2,31 @@
                 </h4>
-                <p class="text-muted mb0" style="margin-top: -5px;">
-                    <span class="o_website_links_short_url text-muted" style="position:relative;">
-                        <span id="o_website_links_host"><t t-esc="widget.link_obj.short_url_host"/></span><span id="o_website_links_code"><t t-esc="widget.link_obj.code"/></span>
-                    </span>
-
-                    <span class="o_website_links_edit_tools" style="display:none;">
-                        <a role="button" class="o_website_links_ok_edit btn btn-sm btn-primary" href="#">ok</a> or
-                        <a class="o_website_links_cancel_edit" href="#">cancel</a>
+                <p class="text-muted mb0">
+                    <span class="o_website_links_short_url d-inline-flex flex-wrap align-items-center border border-primary rounded overflow-hidden">
+                        <span t-esc="widget.link_obj.short_url" class="small text-break px-2 py-1"/>
+                        <button class="btn btn-primary rounded-0 btn_shorten_url_clipboard py-1 w-100 w-md-auto" t-att-data-url="widget.link_obj.short_url">
+                            <small>Copy</small>
+                        </button>
                     </span>
-
-                    <a class="o_website_links_edit_code" aria-label="Edit code" title="Edit code"><span class="fa fa-pencil gray"></span></a>
-
                     <br/>
-                    <span class="badge text-bg-success"><t t-esc="widget.link_obj.campaign_id[1]"/></span>
-                    <span class="badge text-bg-success"><t t-esc="widget.link_obj.medium_id[1]"/></span>
-                    <span class="badge text-bg-success"><t t-esc="widget.link_obj.source_id[1]"/></span>
+                    <span class="badge text-bg-secondary me-1" t-esc="widget.link_obj.campaign_id[1]"/>
+                    <span class="badge text-bg-secondary me-1" t-esc="widget.link_obj.medium_id[1]"/>
+                    <span class="badge text-bg-secondary me-1" t-esc="widget.link_obj.source_id[1]"/>
                 </p>
-                <p class='o_website_links_code_error' style='color:red;font-weight:bold;'></p>
+                <p class="o_website_links_code_error"/>
             </div>
-
-            <div class="col-md-4 col-2">
-                <button class="btn btn-info btn_shorten_url_clipboard mt8">Copy</button>
-                <a role="button" t-attf-href="{{widget.link_obj.short_url}}+" class="btn btn-warning mt8">Stats</a>
+            <div class="col d-flex flex-column align-items-center justify-content-center">
+                <h4 t-esc="widget.link_obj.count"/>
+                <p class="text-muted">clicks</p>
+            </div>
+            <div class="col d-flex align-items-center justify-content-end">
+                <a role="button" t-attf-href="{{widget.link_obj.short_url}}+" class="btn btn-primary">Open</a>
```

Comment:
This URL is weird for the route as usually `+` is used to replace a blank space when doing a search or if the URL requires a space.
Since it's not the subject of the task let's leave it this way then, but it may be something to investigate in the future.